options/linux/generic/utmpx.cpp

```diff
@@ -30,3 +30,8 @@ int utmpxname(const char *) {
 	errno = ENOSYS;
 	return -1;
 }
+
+struct utmpx *getutxid(const struct utmpx *) {
+	// return NULL as musl does
```

Comment:
Maybe add an info log here that we haven't implemented this, as it is a genuinly useful interface.